src/Core/GraphQLOptions.cs

```diff
@@ -18,12 +18,8 @@ public class GraphQLOptions
         /// </summary>
         public bool EnableMetrics { get; set; } = true;
 
-        public INameConverter NameConverter { get; set; }
-
         public Action<UnhandledExceptionContext> UnhandledExceptionDelegate = ctx => { };
 
-        public ISchemaFilter SchemaFilter { get; set; } = new DefaultSchemaFilter();
-
```

Comment:
These removed properties are defined within the schema now, so no need within the options.